src/plugins/daoModule/index.ts

```diff
@@ -0,0 +1,214 @@
+import { getAddress, isAddress } from '@ethersproject/address';
+import { BigNumber, isBigNumberish } from '@ethersproject/bignumber/lib/bignumber';
+import { isHexString } from '@ethersproject/bytes';
+import { Contract } from '@ethersproject/contracts';
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { keccak256 as solidityKeccak256 } from '@ethersproject/solidity';
```

Comment:
It would be better to import `keccak256` from the source like this:
```suggestion
import { keccak256 as solidityKeccak256 } from '@ethersproject/keccak256';
```

Reply:
they are slightly different. The solitidy version performs encoding on top of the hashing, that is especially important in the case where the hash of an array of bytes32 is generated.

Reply:
Ho right, good catch!